lib/core-integration/src/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -1474,6 +1482,130 @@ spec = do
               txs2 <- listTransactions ctx w (Just te) (Just te) Nothing
               length <$> [txs1, txs2] `shouldSatisfy` all (== 0)
 
+    it "TRANS_DELETE_01 - Single Output Transaction" $ \ctx -> do
+        (wSrc, wDest) <- (,) <$> fixtureWallet ctx <*> emptyWallet ctx
+        addrs <- listAddresses ctx wDest
+
+        let amt = 1
+        let destination = (addrs !! 1) ^. #id
+        let payload = Json [json|{
+                "payments": [{
+                    "address": #{destination},
+                    "amount": {
+                        "quantity": #{amt},
+                        "unit": "lovelace"
+                    }
+                }],
+                "passphrase": "cardano-wallet"
+            }|]
+        let (_, feeMax) = ctx ^. feeEstimator $ TxDescription
+                { nInputs = 1
+                , nOutputs = 1
+                }
+
+        r <- request @(ApiTransaction t) ctx (postTxEp wSrc) Default payload
+        verify r
+            [ expectSuccess
+            , expectResponseCode HTTP.status202
+            , expectFieldEqual direction Outgoing
+            , expectFieldEqual status Pending
+            ]
+
+        let txId = getFromResponse #id r
+
+        ra <- request @ApiWallet ctx (getWalletEp wSrc) Default Empty
+        verify ra
+            [ expectSuccess
+            , expectFieldEqual balanceAvailable (faucetAmt - faucetUtxoAmt)
+            ]
+
+        rDel <- request @ApiTxId ctx (deleteTxEp wSrc (ApiTxId txId)) Default Empty
+        expectResponseCode @IO HTTP.status204 rDel
+
+        rb <- request @ApiWallet ctx (getWalletEp wSrc) Default Empty
+        verify rb
+            [ expectSuccess
+            , expectFieldEqual balanceTotal faucetAmt
+            , expectFieldEqual balanceAvailable faucetAmt
+            ]
+
+        rc <- request @ApiWallet ctx (getWalletEp wDest) Default Empty
+        verify rc
+            [ expectSuccess
+            , expectEventually ctx getWalletEp balanceAvailable amt
+            ]
+
+        verify rb
+            [ expectEventually ctx getWalletEp balanceAvailable (faucetAmt - feeMax - amt)
+            ]
+
+        rd <- request @([ApiTransaction t]) ctx (listTxEp wSrc mempty)
+            Default Empty
+        expectResponseCode @IO HTTP.status200 rd
+
+        verify rd
+            [ expectListItemFieldEqual 0 direction Outgoing
+            , expectListItemFieldEqual 0 status InLedger
+            ]
```

Comment:
What about also checking for the `Incoming` transaction on the target wallet :grimacing: ?